src/pkg/packager/deploy.go

```diff
@@ -31,18 +31,14 @@ import (
 )
 
 var (
-	stateInitialized bool
-	hpaModified      bool
-	valueTemplate    *template.Values
-	connectStrings   = make(types.ConnectStrings)
+	hpaModified    bool
+	connectStrings = make(types.ConnectStrings)
```

Comment:
Both of these also are likely not best done here and should likely be put on packager (even though they are less important than state and values they could cause bugs with reuse)

Reply:
(for example hpa modified is never set to false but it is reverted once a package deployment is completed)